user-interface/src/lib/components/CaseNumberInput.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { forwardRef, useImperativeHandle, useRef, useState } from 'react';
+import Input, { InputProps } from './uswds/Input';
+import { InputRef } from '../type-declarations/input-fields';
+
+const BLANK = '';
```

Comment:
We use '' and BLANK in the source. We should pick one for consistency. Probably just drop out BLANK.